packages/jsii-pacmak/test/generated-code/__snapshots__/target-go.test.ts.snap

```diff
@@ -1142,80 +1102,26 @@ import (
 	"github.com/aws/jsii/jsii-calc/go/scopejsiicalcbaseofbase/v2"
 )
 
-// DiamondLeftIface is the public interface for the custom type DiamondLeft
 // Deprecated.
-type DiamondLeftIface interface {
-	GetHoistedTop() string
-	GetLeft() float64
-}
-
-// Deprecated.
-// Struct proxy
 type DiamondLeft struct {
+	// Deprecated.
```

Comment:
Not quite sure how we are getting this double stability banner here, but yeah we shouldn't =)

Reply:
Well - the "member" classes were inconsistent - some emit docs, other don't. An easy fix :)